src/AutoRest.CSharp/Common/Input/Inputs.cs

```diff
@@ -124,6 +124,14 @@ internal record InputParameter(
             ArraySerializationDelimiter: null,
             HeaderCollectionPrefix: null)
         { }
+
+        private readonly Schema? _schema;
+
+        public Schema Schema
```

Comment:
I don't think we want the concept of a Schema in the Input classes.  In the end we will be deleting all of the CodeModel classes and only using the Input classes so any mixing like this makes that final task much more difficult.  There should be a 1 way conversion from CodeModel classes to Input classes.

Reply:
I have removed the Schema here and now I'm using `InputModelType`, thanks for the suggestion.